cue/data/gen.cue

```diff
@@ -66,14 +66,62 @@ Family: scuemata.#Family & {
                     showIn:   number | *0
                 }]
                 // Auto-refresh interval.
-                refresh?: string
+                refresh?: string | false
```

Comment:
Maybe this should be a fix on the API instead of a fix on schema. use a negative or 0 instead of putting a boolean. refresh could be 2 types on the API seems weird.